requirements.txt

```diff
@@ -1,4 +1,4 @@
 numpy>=1.16.0
-pandas>=0.25.0
+pandas>=0.25.0,<1.1.0
```

Comment:
1.1.1 is out. Should see if we can relax this constraint.

Reply:
Upgrading to 1.1.1 results in test failures for `test_daytime_split_day` and `test_daytime_daylight_savings`. The reason for the failures is not immediately clear.

Reply:
`rolling().groupby(center=True)` still broken in pandas > 1.0. https://github.com/pandas-dev/pandas/issues/36040